src/test-support/mocks/src/meshes/_/dataplanes/_/_rules.ts

```diff
@@ -0,0 +1,282 @@
+                  },
+                  matchers: [],
+                  origin: [
+                    {
+                      mesh: 'default',
+                      name: 'default',
+                      type: 'MeshTimeout',
+                    },
+                    {
+                      mesh: 'default',
+                      name: 'override',
+                      type: 'MeshTimeout',
+                    },
+                  ],
+                },
+              ],
+            },
+          ],
+          toRules: [
+            {
+              conf: {
+                connectionTimeout: '2s',
+                idleTimeout: '20s',
+                http: {
+                  requestTimeout: '15s',
+                },
+              },
+              matchers: [
+                {
+                  key: 'kuma.io/service',
+                  not: false,
+                  value: 'foo',
+                },
+              ],
+              origin: [
+                {
+                  mesh: 'default',
+                  name: 'default',
+                  type: 'MeshTimeout',
+                },
+                {
+                  mesh: 'default',
+                  name: 'override',
+                  type: 'MeshTimeout',
+                },
+              ],
+            },
+            {
+              conf: {
+                connectionTimeout: '2s',
+                idleTimeout: '20s',
+                http: {
+                  requestTimeout: '20s',
+                },
+              },
+              matchers: [
+                {
+                  key: 'kuma.io/service',
+                  not: false,
+                  value: 'bar',
+                },
+              ],
+              origin: [
+                {
+                  mesh: 'default',
+                  name: 'default',
+                  type: 'MeshTimeout',
+                },
+                {
+                  mesh: 'default',
+                  name: 'override',
+                  type: 'MeshTimeout',
+                },
+              ],
+            },
+            {
+              conf: {
+                connectionTimeout: '2s',
+                idleTimeout: '20s',
+                http: {
+                  requestTimeout: '10s',
+                },
+              },
+              matchers: [
+                {
+                  key: 'kuma.io/service',
+                  not: true,
+                  value: 'bar',
+                },
+                {
+                  key: 'kuma.io/service',
+                  not: true,
+                  value: 'foo',
+                },
+              ],
+              origin: [
+                {
+                  mesh: 'default',
+                  name: 'default',
+                  type: 'MeshTimeout',
+                },
+              ],
+            },
+          ],
+          type: 'MeshTimeout',
+          warnings: [],
+        },
+      ],
+    } satisfies InspectRulesForDataplane,
+  }
+}
```

Comment:
Let me try if this works out-of-the-box. It’s not terribly important to keep this around, but this is a case where reviewing the result and the code pretty much needs a concrete example that is somewhat appropriate.

Reply:
I registered this under the `test-dataplane` name. I’ll consider adding more examples of this kind for testing purposes, but I’ll do so separately. I half-expect a bit of iteration to be happening on this view going forward.

Reply:
Ah great, glad it works then.

> I’ll consider adding more examples of this kind for testing purposes

Please only do this sort of thing where necessary, testing should ideally uses the tooling we have to do make mocks in the tests. In the past I've found it to only be appropriate in uber complicated setups (which is generally reproducing specific edge case bugs found "in the wild"), those setups were far more complicated that the sort of thing I've seen here up until now.

If we start adding loads of these static examples everywhere it becomes harder and harder to keep them up to date, and then people start testing again static responses again etc etc, and this is what this "dynamic mocks" idea is supposed to avoid.

Reply:
P.S. If you mean "manual" testing purposes thats kinda ok sometimes also.